### PR TITLE
Assorted funcs + Path

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: src/code_F5750.c [hunk 1]
```diff
@@ -142,7 +167,47 @@ void PlayerJump2(ScriptInstance* script, s32 isInitialCall) {
...
+    Bytecode* args = script->ptrReadPos;
+    PlayerStatus* playerStatus = PLAYER_STATUS;
+    f32* t1 = &script->functionTemp[1].f;
+    f32* t2 = &script->functionTemp[2].f;
+    s32* t3 = &script->functionTemp[3].s;
+
+    if (isInitialCall) {
+        Npc** player = &gPlayerNpcPtr;
+
+        (*player)->yaw = playerStatus->targetYaw;
```
Comment (nanaian) [inline src/code_F5750.c:180]: We should rename npc::yaw to npc::targetYaw imo

File: src/code_F5750.c [hunk 2]
```diff
@@ -142,7 +167,47 @@ void PlayerJump2(ScriptInstance* script, s32 isInitialCall) {
     player_jump(script, isInitialCall, 2);
 }
 
-INCLUDE_ASM(s32, "code_F5750", InterpPlayerYaw, ScriptInstance* script, s32 isInitialCall);
+ApiStatus InterpPlayerYaw(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    PlayerStatus* playerStatus = PLAYER_STATUS;
+    f32* t1 = &script->functionTemp[1].f;
+    f32* t2 = &script->functionTemp[2].f;
+    s32* t3 = &script->functionTemp[3].s;
```
Comment (nanaian) [inline src/code_F5750.c:175]: Would be cool to give these clear names.... I'm thinking `initialYaw`, `deltaYaw`, `time` respectively.

File: src/code_FBE10.c [hunk 3]
```diff
@@ -67,7 +67,27 @@ ApiStatus func_802D7B74(ScriptInstance* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "code_FBE10", func_802D7BA4);
 
-INCLUDE_ASM(s32, "code_FBE10", func_802D7CF8);
+ApiStatus func_802D7CF8(ScriptInstance* script, s32 isInitialCall) {
```
Comment (nanaian) [inline src/code_FBE10.c:70]: Can call this `Spawn802D9D50`; it's equivalent to the script `spawn D_802D9D50` except variables 7..=16 aren't copied.

File: src/code_FBE10.c [hunk 4]
```diff
@@ -67,7 +67,27 @@ ApiStatus func_802D7B74(ScriptInstance* script, s32 isInitialCall) {
...
+    Bytecode* args = script->ptrReadPos;
+    s32 var0 = get_variable(script, *args++);
+    s32 var1 = get_variable(script, *args++);
+    s32 var2 = get_variable(script, *args++);
+    s32 var3 = get_variable(script, *args++);
+    s32 var4 = get_variable(script, *args++);
+    s32 var5 = get_variable(script, *args++);
+    s32 var6 = get_variable(script, *args++);
+    ScriptInstance* newScript = start_script(&D_802D9D50, 1, 0);
+
```
Comment (nanaian) [inline src/code_FBE10.c:80]: Probably move the `start_script` to this line and have the empty line between the `get_variables` and it imo

File: src/code_F5750.c [hunk 5]
```diff
@@ -142,7 +167,47 @@ void PlayerJump2(ScriptInstance* script, s32 isInitialCall) {
...
+        }
+        if (*t2 > 180.0f) {
+            *t2 -= 360.0f;
+        }
+    }
+
+    if (*t3 > 0) {
+        Npc** player = &gPlayerNpcPtr;
+
+        (*player)->duration++;
```
[the remark this answers is not in the document]
Comment (ethteck) [reply src/code_F5750.c:197]: I assume this is some sort of macro stuff, or we have something wrong about gPlayerNpcPtr. We'll learn more as we decomp more stuff that uses it.

File: src/code_F5750.c (hunk 1 above)
Comment (ethteck) [reply src/code_F5750.c:180]: Are you sure it represents a target value and not a current value?

Comment (nanaian) [reply src/code_F5750.c:180]: IIRC clover mentioned it to me but I could be wrong

**Review by nanaian — APPROVED** [review]

File: src/code_FBE10.c (hunk 4 above)
Comment (pixel-stuck) [reply src/code_FBE10.c:79]: I agree with this; maybe I missed some conversation on discord, since it was marked as resolved

Comment (nanaian) [reply src/code_FBE10.c:79]: Yeah, it was changed in e8ba79a88c5e2d52bc34bc3e0bff98e1ee37fc11

**Review by pixel-stuck — APPROVED** [review]

**Pull request merged**